src/jobs/qa-automation-suite.yaml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  test_path:
```

Comment:
I'm not sure I understand what this parameter does.

Reply:
It's the path to the test suite to run, which is locate inside the docker image

Reply:
so we have a fixed number of possibilities?
can we list them, at least in a comment?

Reply:
I can provide the list:

Dev -> Suite/API/DEV.xml,Suite/WEB/DEV.xml,Suite/WEB/DEV2.xml,Suite/E2E/DEV.xml
Staging -> Suite/API/DEV.xml,Suite/WEB/STAGING.xml,Suite/WEB/STAGING.xml,Suite/E2E/STAGING.xml
Prod -> Suite/API/PROD.xml,Suite/WEB/PROD.xml,Suite/E2E/PROD.xml

Reply:
New suites can be added, it seems too cumbersome to add them here everytime there's a change in the qa-automation repo.

Reply:
What I suggest doing is adding a link to the qa-automation repo, where all files are located

Reply:
I agree into keeping this list in the `qa-automation-test` repository, and here we keep a link.
Works for you @jonathancarvalheiro ?

Reply:
Seems good, or directly to this link: https://handbook.dev.codacy.org/product/engineering/QA/feature-test-process.html#suites-to-run-for-each-environment, what you think?

Reply:
up to you, either way I'm good with it  :+1: